.github/renovate.json5

```
@@ -1,20 +1,22 @@
 {
   // Configuration file for RenovateBot: https://docs.renovatebot.com/configuration-options
-  extends: ["config:base"],
+  extends: ["config:recommended"],
```

The `semanticCommitType: "build"` on line 153 only worked for a month or so before renovate changed something.  

The [docs](https://docs.renovatebot.com/semantic-commits/#changing-the-semantic-commit-type) suggest this:

```suggestion
  extends: ["config:recommended", ":semanticCommitTypeAll(build)"],
```

WDYT about sneaking this into your PRs?

---

Ooh, that's great! Thanks!